### DISCUSSION TITLE
toggle

### DISCUSSION THREAD

**Discussion by polomint278** [zauberzeug/nicegui#4160] [Ideas / Feature Requests]
ui.toggle seems to be based on
https://quasar.dev/vue-components/button-toggle

is there any plans to allow
https://quasar.dev/vue-components/toggle/

as the toggle is smaller and would fit my case better.

Currently im using the simple
```
ui.toggle({False: "Not Monitored", True: "Monitored"}, value=item['monitor_status'],
                              on_change=lambda e, i=i: toggle_monitored(i))
```

**Comment by rodja** [comment]
Duplicate of https://github.com/zauberzeug/nicegui/discussions/4159
